src/condor_ce_trace

```diff
@@ -205,50 +205,50 @@ def check_job_submit(job_info, schedd_ad):
 
     try:
         schedd = htcondor.Schedd(schedd_ad)
-    except RuntimeError, e:
+    except RuntimeError as e:
         raise ce.CondorRunException("Failed to contact schedd at %s due to the following error:\n%s" \
                                     % (schedd_ad['Machine'], e))
-    print "Submitting job to schedd %s" % schedd_ad['MyAddress']
+    print(f"Submitting job to schedd {schedd_ad['MyAddress']}")
     ad_results = []
     try:
         cluster = schedd.submit(job_ad, 1, True, ad_results)
-    except RuntimeError, e:
+    except RuntimeError as e:
         raise ce.CondorRunException("- Failed to submit job to %s due to the following error:\n%s" \
                                     % (schedd_ad['Machine'], e))
 
-    print "- Successful submission; cluster ID %d" % cluster
+    print(f"- Successful submission; cluster ID {cluster}")
 
-    print "Resulting job ad:", ad_results[0]
+    print(f"Resulting job ad: {ad_results[0]}")
 
-    print "Spooling cluster %d files to schedd %s" % (cluster, schedd_ad['MyAddress'])
+    print(f"Spooling cluster {cluster} files to schedd {schedd_ad['MyAddress']}")
     try:
         schedd.spool(ad_results)
-    except RuntimeError, e:
+    except RuntimeError as e:
         raise ce.CondorRunException("- Failed to spool files to %s due to the following error:\n%s" \
                                  % (schedd_ad['Machine'], e))
 
-    print "- Successful spooling"
+    print("- Successful spooling")
 
     attempts = int(htcondor.param.get("CONDOR_CE_TRACE_ATTEMPTS", 600))
     last_status = -1
     for attempt in range(attempts):
         if G_DEBUG:
-            print "Querying job status (%d/%d)" % (attempt+1, attempts)
+            print(f"Querying job status ({attempt+1}/{attempts})")
         try:
             ad = schedd.query("ClusterID == %d" % cluster, ["JobStatus", "ClusterID", "ProcID"])
-        except RuntimeError, e:
+        except RuntimeError as e:
             raise ce.CondorRunException("- Failed to query job status due to the following error:\n%s" %  e)
         if len(ad) != 1:
             raise ce.CondorRunException("Could not find the job in cluster %d" % cluster)
         status = ad[0]['JobStatus']
         if G_DEBUG:
-            print "Job status:", CONDOR_JOB_STATUS.get(status, status)
+            print(f"Job status: {CONDOR_JOB_STATUS.get(status, status)}")
         elif last_status != status:
             if last_status == -1:
-                print "Job status:", CONDOR_JOB_STATUS.get(status, status)
+                print(f"Job status: {CONDOR_JOB_STATUS.get(status, status)}")
             else:
-                print "Job transitioned from", CONDOR_JOB_STATUS.get(last_status, last_status), \
-                    "to", CONDOR_JOB_STATUS.get(status, status)
+                print(f"Job transitioned from {CONDOR_JOB_STATUS.get(last_status, last_status)} \
+                    to {CONDOR_JOB_STATUS.get(status, status)}")
```

Comment:
```suggestion
                print(f"Job transitioned from {CONDOR_JOB_STATUS.get(last_status, last_status)}"
                      f"to {CONDOR_JOB_STATUS.get(status, status)}")
```
the backslash version would have included the indent of the 'to' line in the output.